packages/google_maps_flutter/google_maps_flutter_web/pubspec.yaml

```diff
@@ -23,7 +23,8 @@ dependencies:
   flutter_web_plugins:
     sdk: flutter
   google_maps: ^7.1.0
-  google_maps_flutter_platform_interface: ^2.5.0
+  google_maps_flutter_platform_interface: ^2.6.0
+  js: ^0.6.7
```

Comment:
On version 0.5.5 we removed the `package:js` dependency so this package could compile to Wasm, please don't add it back! There's a new (better) way of doing js-interop.